app/constants/view.ts

```diff
@@ -18,6 +18,10 @@ export const HEADER_WITH_SUBTITLE = 24;
 export const IOS_HEADER_SEARCH_INSET = 20;
 export const TABLET_HEADER_SEARCH_INSET = 28;
 export const ANDROID_HEADER_SEARCH_INSET = 11;
+export const LARGE_HEADER_PROGRESSVIEW_OFFSET = Platform.select({
+    android: LARGE_HEADER_TITLE + ANDROID_DEFAULT_HEADER_HEIGHT + HEADER_WITH_SUBTITLE,
+    ios: LARGE_HEADER_TITLE + IOS_DEFAULT_HEADER_HEIGHT + HEADER_WITH_SUBTITLE,
+});
```

Comment:
what is he difference between this and the value of `scrollPaddingTop` that you get from the collapsible hook? I understand that you are using for the spacing of the refresh indicator, but the padding of the scroll should be sort of the same, right ?